presto-hive/src/main/java/com/facebook/presto/hive/clustering/HiveClusteringFunction.java

```diff
@@ -0,0 +1,64 @@
+package com.facebook.presto.hive.clustering;
+
+import com.facebook.presto.common.Page;
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.hive.BucketFunctionType;
+import com.facebook.presto.hive.HiveBucketFunction;
+import com.facebook.presto.hive.HiveType;
+import com.facebook.presto.spi.BucketFunction;
+import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.facebook.presto.hive.BucketFunctionType.HIVE_CLUSTERING;
+import static com.facebook.presto.hive.BucketFunctionType.HIVE_COMPATIBLE;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.Objects.requireNonNull;
+
+public class HiveClusteringFunction implements BucketFunction
```

Comment:
move `implements BucketFunction` to a new line with 8 spaces as the indentation.